DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: rextendr
 Title: Call Rust Code from R using the 'extendr' Crate
-Version: 0.3.1.9000
```

Comment:
We have snapshot tests that depend on this version (yep, not the best setup right now). If you want to bump it, please do update the snapshots as well.

Reply:
pronto!